testsuite/views.py

```diff
@@ -117,6 +120,19 @@ def health_summary():
     return render_template('health_summary.html', vendors=vendors, latest_results=latest_results)
 
 
+@app.route('/update_bloom', methods=['GET'])
+def update_bloom_filter():
+
+    bloom_file = './data/codes.bf'
+
+    r = requests.get(current_app.config['BLOOM_FILTER_URL'])
+    open(bloom_file, 'wb').write(r.content)
```

Comment:
Just an allocation nit: can you use a `with` block to make it clear there's a `close` happening after the write?  (https://stackoverflow.com/a/5362917 for example discusses)